readthedocs/core/management/commands/reindex_elasticsearch.py

```diff
@@ -41,10 +45,8 @@ def handle(self, *args, **options):
             except:
                 # This will happen on prod
                 commit = None
+
             try:
-                page_list = parse_json.process_all_json_files(version, build_dir=False)
-                index_search_request(
-                    version=version, page_list=page_list, commit=commit,
-                    project_scale=0, page_scale=0, section=False, delete=False)
```

Comment:
We could pretty easily add a delete param to the update_search task, which we could use here, and would solve this problem.